static/js/message_list_data.js

```diff
@@ -178,6 +178,18 @@ export class MessageListData {
     }
 
     messages_filtered_for_user_mutes(messages) {
+        if (this.filter === undefined || this.filter.is_non_huddle_pm === undefined) {
```

Comment:
is `is_non_huddle_pm` a function or a property?

Reply:
It is a function. I've removed this check now, and fixed a few tests to use real `Filter` objects instead.